app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    words = command.split(" ")
+    command_name = words[0]
```

Comment:
if len(command.split()) == 3:
        command_name, name_to_copy, new_file_name = command.split()

Reply:
ah, so this is the way

Reply:
I was trying to avoid double if